care/facility/migrations/0125_auto_20200626_0146.py

```diff
@@ -0,0 +1,19 @@
+# Generated by Django 2.2.11 on 2020-06-25 20:16
+
+from django.db import migrations, models
+import uuid
```

Comment:
Codacy found an issue: [standard import "import uuid" should be placed before "from django.db import migrations, models"](https://app.codacy.com/gh/coronasafe/care/pullRequest?prid=5811718)